docs/manipulators/manipulators.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Manipulators Overview
```

Comment:
Agreed that Gen3 lite is the only manipulator supported with PACS. Do we plan to list other common manipulators that are "supported" by CPR even though they don't have PACS integration?

Reply:
I want to add some content about UR3e and UR5e. We already have a PACS brackets for them. We also sell a UR5e remote integration kit, so a customer can add a UR5e to their existing Husky. I think we should add the remote integration instructions to docs.clearpathrobotics.com.